src/openvic-simulation/economy/production/ResourceGatheringOperation.hpp

```diff
@@ -8,8 +8,12 @@
 #include "openvic-simulation/utility/Getters.hpp"
 
 namespace OpenVic {
+	struct ProvinceInstance;
+
 	struct ResourceGatheringOperation {
 	private:
+		ModifierEffectCache const& modifier_effect_cache;
```

Comment:
`ModifierEffectCache` could also be forward declared in this file, and then only included in `ResourceGatheringOperation.cpp`